app/main/forms/brief_forms.py

```diff
@@ -47,12 +50,12 @@ def get_briefs(self):
         else:
             statuses = [id_ for id_, label in self.status.choices]
         if 'closed' in statuses:
-            statuses.append('awarded')
+            statuses.extend(CLOSED_BRIEF_STATUSES)
```

Comment:
Feels a little odd adding 'closed' only to pull it out later.
Could go for something like:
```
if 'closed' in statuses:
    # Add hidden closed statuses
    statuses.extend(['awarded', 'cancelled', 'unsuccessful'])
```
Not _that_ bothered but it was a bit hard to read on the first pass

Reply:
I think this problem boils down to me being unable to think of a good name for the group of statuses `awarded, cancelled, unsuccessful`. How about I change the list to be `BRIEF_AWARD_STATUSES` and remove `closed`?

Hopefully this hack won't be needed if/when we start filtering by the statuses properly in the View Opportunities page (rather than just treating them as closed).